client/python/tensorflow-serving-api/samples/README.md

```diff
@@ -217,7 +218,8 @@ Classification accuracy: 100.00
 
 Using binary inputs feature requires loading model with layout set to `--layout NHWC:NCHW`:
 ```bash
-docker run -d --rm -e "http_proxy=$http_proxy" -e "https_proxy=$https_proxy" -p 8000:8000 -p 9000:9000 openvino/model_server:latest --model_name resnet --model_path gs://ovms-public-eu/resnet50-binary --port 9000 --rest_port 8000 --layout NHWC:NCHW
+wget -N https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.{xml,bin} -P models/resnet50/1
```

Comment:
Why `-N` is not added to all of commands?

Reply:
-N is downloading the file only when new version is available, it will avoid duplicates when command is executed mulitple times

Reply:
-N is not used in all wget lines you add

Reply:
in the first execution on page -N is not needed